frappe/utils/remote_migrations.py

```diff
@@ -0,0 +1,267 @@
+# imports - standard imports
+import getpass
+import json
+import re
+import sys
+
+# imports - third party imports
+import click
+from html2text import html2text
+import requests
+
+# imports - module imports
+import frappe
+import frappe.utils.backups
+from frappe.utils import get_installed_apps_info
+from frappe.utils.commands import render_table, padme
+
+
+def get_new_site_options():
+	site_options_sc = session.post(options_url)
+
+	if site_options_sc.ok:
+		site_options = site_options_sc.json()["message"]
+		return site_options
+	else:
+		print("Couldn't retrive New site information: {}".format(site_options_sc.status_code))
+
+
+def is_valid_subdomain(subdomain):
+	if len(subdomain) < 6:
+		print("Subdomain too short. Use 5 or more characters")
```

Comment:
Did you noticed, Conditions & Message?

Image used description says, enter subdomain, you inserted 'tally' and then, what validation message says, "Use 5 or more characters" while 'tally' has already 5 characters!